### ISSUE TITLE
Penalties System for Flag

### ISSUE THREAD

**Issue by olivierd1** [moloch--/RootTheBox#467]
Hi! Is it planned to have the penalties system for each flag instead of a global configuration ?

**Comment by jeffg2k** [comment]
No, that hasn't been requested yet.  Currently, the penalty is a percentage of the flag value, so works pretty well across difficulty levels if it's enabled.  I think for most games, penalties are not used, but it's good for learning environments so people don't brute force.  I could see a use case where it's only applied to multiple choice or something.  What's your use case?  I probably won't put it high on the priority list, but I'm interested to hear what you're thinking about.

**Comment by olivierd1** [comment]
My use case is for a set of challenges (jeopardy type) where flags are subject to brute force (flags are simple words) and not really a CTF where teams compete each other. I'd like to set a high penalty for flags requiring only a common word and another amount of penalty for flags a little more difficult (date for example)
